polaris-core/src/main/java/org/apache/polaris/core/persistence/PolarisEntityManager.java

```diff
@@ -40,39 +39,28 @@
  */
 public class PolarisEntityManager {
   private final PolarisMetaStoreManager metaStoreManager;
-  private final EntityCache entityCache;
+  private final ResolverFactory resolverFactory;
 
   // Lazily instantiated only a single time per entity manager.
   private ResolvedPolarisEntity implicitResolvedRootContainerEntity = null;
 
   /**
    * @param metaStoreManager the metastore manager for the current realm
-   * @param entityCache the entity cache to use (it may be {@code null}).
+   * @param resolverFactory the resolver factor to use
    */
   public PolarisEntityManager(
-      @Nonnull PolarisMetaStoreManager metaStoreManager, @Nullable EntityCache entityCache) {
+      @Nonnull PolarisMetaStoreManager metaStoreManager, @Nonnull ResolverFactory resolverFactory) {
```

Comment:
Nit:
```suggestion
      @Nonnull PolarisMetaStoreManager metaStoreManager, ResolverFactory resolverFactory) {
```

We kind-of agreed that parameters a not-null by default. But I don't mind having it explicit.